src/components/code-line.component.js

```diff
@@ -130,6 +130,7 @@ export class CodeLine extends Component {
                 CodeTag={Text}
                 codeTagProps={{ style: styles.codeLine }}
                 customStyle={customStyle}
+                fontFamily={fonts.fontCode.fontFamily}
```

Comment:
Oh man thank you, took a little bit of tinkering with and you still made it happen.

Not a problem _at all_ if we just have to add a separate prop for `fontFamily` :)